zkevm-circuits/src/state_circuit/constraint_builder.rs

```diff
@@ -234,10 +237,92 @@ impl<F: Field> ConstraintBuilder<F> {
         // TODO: Missing constraints
     }
 
+    fn build_tx_log_constraints(&mut self, q: &Queries<F>) {
+        self.require_equal(
+            "is_write is always true for TxLog",
+            q.is_write.clone(),
+            1.expr(),
+        );
+
+        // removed following field_tag-specific constraints as issue
+        // https://github.com/privacy-scaling-explorations/zkevm-specs/issues/221
```

Comment:
```suggestion
        // Comment out the following field_tag-related constraints as it is duplicated between state circuit and evm circuit.
        // For more information, please refer to https://github.com/privacy-scaling-explorations/zkevm-specs/issues/221
```

Reply:
update in [1640863](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/552/commits/1640863d35db598f19b60f291d1cb09f98c6d264) thanks for improvement !